docs/host_binaries_solutions_comparison.md

```diff
@@ -0,0 +1,262 @@
+# Host binaries solutions comparison
+
+## tl;dr
+
+After research and test git-annex, we think it's not suitable for our purposes. We try to use GitHub Releases, documentation and automating process.
+
+|                   |           GIT LFS             | GIT SUBMODULES | GIT ANNEX                    |
+|:-----------------:|:-----------------------------:|:--------------:|:----------------------------:|
+|       Costs       |   5$/month per pack (50gb)    |   Cannot find clear answer            | Depend on storage provider   |
+| Versioning Files  |           Yes                 |   Yes          |  Yes                         |
+| Branch support    |           Yes                                             |   No, they are just a pointer to a particular commit of the submodules repository. It's can be harder to manage than git lfs and git-annex. We shouldn't store binary files in git because it's not designed for it. This is a purpose why git-annex and git-lfs exists. Some about it [here](https://robinwinslow.uk/2013/06/11/dont-ever-commit-binary-files-to-git/#:~:text=It%27s%20important%20to%20never%20commit,size%20will%20still%20be%20large.) |  Yes |
+| Flexibility       |  Yes, you can set your lfs server. [Check here](https://github.com/git-lfs/git-lfs/wiki/Implementations)   |   No | Yes, can use different storage providers. |
+
+By comparing these three solutions and talk with @jan-gogo  in our opinion the best solution is git-annex because:
```

Comment:
It's not actual. Currently, we are staying with GitHub releases, we need to automate release process.

Reply:
Thanks, changed!